src/bake/DevServer.zig

```diff
@@ -6859,6 +6920,13 @@ fn releaseRelativePathBuf(dev: *DevServer) void {
     }
 }
 
+/// Either of two conditions make this true:
+/// - The inspector is enabled
+/// - The user passed "console": true in serve({development: {console: true}}) options
+fn shouldReceiveConsoleLogFromBrowser(dev: *const DevServer) bool {
+    return dev.inspector() != null or dev.broadcast_console_log_from_browser_to_server;
```

Comment:
can `dev.inspector() != null` change after the DevServer is initialized? if so, there will need to be a notification for when this changes so that all client bundles can be cache-busted. it would be ok to make that a TODO comment somewhere.

If not, then this can be precomputed into the boolean field at startup.

Reply:
Added a comment